addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4335,6 +4322,7 @@ export class OdooEditor extends EventTarget {
                     star.classList.toggle('fa-star-o', true);
                     star.classList.toggle('fa-star', false);
                 };
+                ev.preventDefault();
```

Comment:
Why?

Reply:
Without the `preventDefault`, it saves changes made in the star element even without calling `_onReadonlyClickStar`. Initially, I followed the same approach for the checklist, and it worked; removing `preventDefault` for the checklist saved it even if `_onReadonlyClickChecklist` was not called.

However, after reading DMO's log note, I discovered that there's already a function that can save the checklist when first clicked. So, I revised my fix.
and i have removed `preventDefault` here to avoid potential future issues because of this.